dimagi/pages/templates/pages/home.html

```diff
@@ -2,10 +2,6 @@
 {% load i18n %}
 {% load dimagidotcom %}
 
-{% block top_banner %}
```

Comment:
I think this is the only thing that needs to be removed. let's keep the things in place in case we want to add another banner in the future.

Reply:
@biyeun Created a new PR for this issue you can delete this PR/branch. Thanks